src/components/TaskCreation.js

```diff
@@ -20,34 +21,48 @@ import { collection,
 const TaskCreation = (props) => {
     const [TaskTitle, setTaskTitle] = useState('');
     const [TaskDescription, setTaskDescription] = useState('');
-    const [PointValue, setPointValue] = useState('15');
+    const [PointValue, setPointValue] = useState('0');
     const roomID = props.roomID;
     const taskCollectionRef = collection(db, 'rooms', roomID, 'tasks');
     const toast = useToast();
     const time = new Date();
     const [error, setError] = useState('');
+    const [selectedTaskType, setSelectedTaskType] = useState('');
 
+    //stores task description
     const handleDescriptionChange = (event) => {
         setTaskDescription(event.target.value);
     }
 
+    //stores task title
     const handleTitleChange = (event) => {
         setTaskTitle(event.target.value);
     }
 
+    //stores point value
     const handlePointChange = (value) => {
         setPointValue(value);
     }
 
+    //stores task type
+    const handleChangeTaskType = (event) => {
+        setSelectedTaskType(event.target.value);
+        console.log(selectedTaskType);
+    }
+
+    //handles task submisison
     const handleAddTask = async () => {
         let newTask = {
             title: TaskTitle,
             description: TaskDescription,
             pointValue: PointValue,
+            taskType: selectedTaskType,
             dateCreated: time.toLocaleTimeString([], {hour: '2-digit', minute: '2-digit'}),
             isComplete: false,
             completedBy: []
         };
+
+        //error handling for blank title
         if (newTask.title === '') {
             setError("Task title cannot be blank");
             toast ({
```

Comment:
This toast logic takes up a lot of lines and it seems like some of the values are the same every time. Can we pull it out into a separate function in this file to keep code concise? Looks like only title, description, and status actually change among your four calls to toast here